src/toil/jobStores/fileJobStore.py

```diff
@@ -187,6 +193,44 @@ def fileExists(self, jobStoreFileID):
             raise NoSuchFileException("Path %s is not a file in the jobStore" % jobStoreFileID)
         return True
 
+    fileRegex = re.compile("[fF]ile://.+")
+
+    def importFile(self, sourceUrl, jobStoreID=None):
+        url = urlparse.urlparse(sourceUrl)
+        if self.fileUrlRegex.match(sourceUrl):
+            localFilePath = url.netloc + url.path
+
+            fd, absPath = self._getTempFile(jobStoreID)
+            shutil.copyfile(localFilePath, absPath)
+            os.close(fd)
+
+            jobStoreFileID = jobStoreID or self._getRelativePath(absPath)
+            self._assertFileMatchesJobStoreFileID(jobStoreFileID, localFilePath)
+            return jobStoreFileID
+        else:
+            raise RuntimeError("The url '%s' is not a valid file or s3 url" % sourceUrl)
```

Comment:
Can you import from s3 to the fileStore? If not I don't think it should be mentioned in the error